Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Scuba diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "You know, I recently thought about water sports that we could do. How about diving?"
+    m 3eub "By the way [player], have you heard about scuba diving?"
```

Comment:
You already set the stage for the topic with the first line, so you don't need this one anymore. 

Instead, I would have a line briefly outlining why she chose to think about diving in particular to more smoothly lead into the topic.

Reply:
Do you mean to remove line 2?

Reply:
Yes, this second line isn't needed, but what you could replace it with could be a line giving you an idea why she thought about scuba diving in particular.